src/NuGet.Services.Metadata.Catalog.Monitoring/Utility/JsonSerializerUtility.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using NuGet.Protocol;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    public static class JsonSerializerUtility
+    {
+        /// <summary>
+        /// The <see cref="JsonSerializerSettings"/> to use.
+        /// </summary>
+        public static JsonSerializerSettings SerializerSettings => _serializerSettings.Value;
+        private static Lazy<JsonSerializerSettings> _serializerSettings = new Lazy<JsonSerializerSettings>(() =>
```

Comment:
Why lazy?

Reply:
No huge reason, just thought it was prettier syntactically than the other options.

Reply:
I agree it's prettier syntactically, but, lazy has thread-safety overhead. I would use a static constructor instead.